src/backend/api/v1/cost/views.py

```
@@ -6,35 +6,22 @@
+from backend.apps.clusters.models import SubscriptionCost
 
 
 class CostView(AdminOnlyViewSet, mixins.ListModelMixin, mixins.CreateModelMixin, GenericViewSet):
     serializer_class = CostSerializer
 
-    def get_queryset(self) -> QuerySet[Costs]:
-        return Costs.objects.all()
+    def get_queryset(self) -> QuerySet[SubscriptionCost]:
+        return SubscriptionCost.objects.all()
+
+    def list(self, request, *args, **kwargs) -> Response:
```

Ruff (0.15.2)

[warning] Unused method argument: `request`

(ARG002)